Fitness/Controllers/ClassListViewController.swift

```diff
@@ -332,7 +361,13 @@ extension ClassListViewController: UISearchBarDelegate {
         }
         filteringIsActive = true
         filteredClasses = classList[calendarDatesList.firstIndex(of: calendarDateSelected)!]
-        filteredClasses = filteredClasses.filter { $0.className.lowercased().contains(searchText.lowercased())}
+        filteredClasses = filteredClasses.filter {
+            $0.className.lowercased().contains(searchText.lowercased()) ||
```

Comment:
note to self, if we actually use search, we should override the description var on the class struct and then have it return one string that we can do `$0.description.contains(...)` and the description has className, classDescription, classLocation etc